website/content/docs/secrets/pki/index.mdx

```diff
@@ -8,13 +8,14 @@ description: The PKI secrets engine for Vault generates TLS certificates.
 
 @include 'x509-sha1-deprecation.mdx'
 
-!> **Consul users:** Do not use [Vault
+!> **Vault 1.11.0+ incompatible as Consul CA provider:** Do not use [Vault
 v1.11.0+](/vault/docs/release-notes/1.11.0#known-issues) as Consul’s Connect CA
 provider — the intermediate CA will become unable to issue the leaf nodes
 required by service mesh and all actions (if using auto-encrypt or auto-config,
-and using mTLS for server-to-server communication). Refer to the [Knowledge Base
-article]() for more information about the underlying cause and recommended
-workaround.
+and using mTLS for server-to-server communication). If you are already using
+Vault 1.11.0+ as a Connect CA, refer to the [Knowledge Base
+article](https://support.hashicorp.com/hc/en-us/articles/11308460105491) for
+more information about the underlying cause and recommended workaround.
```

Comment:
I proposed a wording change on the Consul equivalent of this notification here: https://github.com/hashicorp/consul/pull/15422#pullrequestreview-1184621413

Once @im2nguyen resolves that comment and merges the Consul docs PR, you could update this PR to use the same wording.

Reply:
The wording I see here currently is not the wording of what was recently merged in the Consul docs. The Consul docs PR wording was updated within the past hour.

I'll propose a change that aligns with the final Consul wording.